src/components/js/Todos.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import '../css/Todos.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+
+export default function Todos(props) {
+  const { handleRemove, storage, todoStatus } = props;
+  const editTodo = (e) => {
+    e.target.readOnly = false;
+  };
+  const addReadOnly = (e) => {
+    e.target.readOnly = true;
+  };
+  return (
+    storage.map((element) => (
+      <li className="todo-item" id={element.id} key={element.id}>
+        <ul className="todo-item-inner">
+          <li className={`${element.completed ? 'checked' : ''}`}>
+            <input className="todo-detail todo-checkbox" type="checkbox" onChange={() => todoStatus(element.id)} checked={element.completed} />
+            <input className="todo-detail todo-value" type="text" onBlur={addReadOnly} onClick={editTodo} onChange={(event) => todoStatus(element.id, event.target.value)} defaultValue={element.title} readOnly />
+          </li>
+          <li className="trash-icon-container">
+            <FontAwesomeIcon icon={faTrash} id={element.id} onClick={handleRemove} />
+          </li>
+        </ul>
```

Comment:
- [ ] It would look better if all todo elements are aligned correctly. so make use to align the check box with the text.
![image](https://user-images.githubusercontent.com/42470159/154815328-d95f3a3d-9d86-4274-9307-fd22b75c39bc.png)

Reply:
It looks good on my side